.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,34 @@
+name: frontend-shared package
+on:
+  release:
+    types:
+      - published
+jobs:
+  update-frontend-shared-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
You might as well use the latest Node version for this task. I can't think of any features that we immediately _require_ from it though, so skip this change if it is difficult. You'll have to change the `node12` config in `action.yml` as well.

Reply:
sure, I think this was just copy pasted from examples. :)

Reply:
No go on node15 :(

Reply:
No problem. We can stick with Node 12 for now.